src/components/Listing/Listing.js

```diff
@@ -1,6 +1,6 @@
 import { mapMutations } from 'vuex'
 import { router } from '../../router/index.js'
-import { normalizeImageUrl } from '../../helpers/utils'
+import utils from '../../helpers/utils.js'
```

Comment:
If you are using only the `normalizeImageUrl` function from utils, I consider a good practice import only `normalizeImageUrl`.

Reply:
Ive tried to do
`import { normalizeImageUrl } from '../../helpers/utils.js'`
and
`import normalizeImageUrl from '../../helpers/utils.js'`
with `export default` in `utils.js` but none worked

Reply:
done on another branch he he